Kudu.Core/Jobs/ContinuousJobLogger.cs

```diff
@@ -32,6 +32,8 @@ public ContinuousJobLogger(string jobName, IEnvironment environment, ITraceFacto
             ResetLockedStatusFile();
         }
 
+        public event Action RolledLogFile;
```

Comment:
I used Action because I'd be ignoring the object/sender callback params of EventHandler